packages/editor-ui/src/modules/workflows.ts

```diff
@@ -0,0 +1,34 @@
+
+import { renameWorkflow } from '@/api/workflows';
+import { ActionContext, GetterTree, Module } from 'vuex';
+import {
+	IRootState,
+	ITag,
+	IWorkflowsState,
+} from '../Interface';
+
+const module: Module<IWorkflowsState, IRootState> = {
+	namespaced: true,
+	state: {},
+	getters: {
+		currentWorkflowTagIds: (state: IWorkflowsState, getters: GetterTree<IWorkflowsState, IRootState>, rootState: IRootState): string[] => {
+			const tags = (rootState.workflow.tags || []) as string[];
+			// @ts-ignore
+			const tagsMap = rootState.tags.tags;
```

Comment:
`tags` property does not exist on `IRootState`. Maybe this should be typed in the interface instead of ignored? This would also make it easier to follow what is going on at the return statement.

Reply:
I experimented a bit with decorators and it seemed to work but it required a few changes that were out of scope https://github.com/championswimmer/vuex-module-decorators/
I think this is best handled separately